test/unit/spec/localparticipant.js

```diff
@@ -122,19 +122,16 @@ describe('LocalParticipant', () => {
     'unpublishTracks',
   ].forEach(method => {
     describe(`#${method}`, () => {
-      const trackMethod = method.slice(0, -1);
+      const trackMethod = method.match('(add|publish).*') ? 'publishTrack' : 'unpublishTrack';
```

Comment:
I think the former (`slice(0, -1)`) is actually a little easier to read than the `match`; however, I also see you've changed the result of `trackMethod` here: "addTracks" and "removeTracks" now map to "publishTrack' and "unpublishTrack" instead of "addTrack" and "removeTrack". Is this intentional? Also, is this change necessary?